web/onos-gui/src/app/onos-topo/proto/onos-topo-device.service.ts

```diff
@@ -45,7 +45,7 @@ export class OnosTopoDeviceService {
         const listRequest = new ListRequest();
         listRequest.setSubscribe(subscribe);
         const stream = this.deviceServiceClient.list(listRequest, {
-            Authorization: 'Bearer ' + this.loggedinService.idToken,
+            // Authorization: 'Bearer ' + this.loggedinService.idToken,
```

Comment:
probably should leave this in

Reply:
oh yes, sorry I always forget to uncomment this bit